### ISSUE TITLE
Kotlin DSL Builder version

### ISSUE THREAD

**Issue by WesJD** [WesJD/AnvilGUI#302]
<sup>Originally posted by **Pseudow** November  1, 2023</sup>
Add kotlin DSL option would be so good</div>

**Comment by 0dinD** [comment]
I don't understand what this issue is about? Is it just a documentation issue (only Maven instructions are provided right now)? Because you can already use AnvilGUI from a Gradle project (Groovy or Kotlin).

**Comment by portlek** [comment]
> I don't understand what this issue is about? Is it just a documentation issue (only Maven instructions are provided right now)? Because you can already use AnvilGUI from a Gradle project (Groovy or Kotlin).

it's not about using it in kotlin environment, DSL is a thing that you implement in a library to give better support for kotlin. people can use the library with kotlin. DSL builder is a thing that you implement to make the kotlin users easier to use the existing library.

**Comment by 0dinD** [comment]
Yeah I was just about to write that I now realize they were talking about the `AnvilGUI.Builder` class. I thought they meant Gradle when they just said "Kotlin DSL".
